### PR TITLE
fixed discovery rule and hostgroup tests

### PR DESCRIPTION
**Pull request by sghai**

### CODE REVIEW

File: tests/foreman/cli/test_discoveryrule.py [hunk 1]
```diff
@@ -262,7 +262,7 @@ def test_negative_create_with_too_long_limit(self):
         created
         """
         with self.assertRaises(CLIFactoryError):
-            self._make_discoveryrule({u'host-limit': '9999999999'})
+            self._make_discoveryrule({u'hosts-limit': '9999999999'})
```
Comment (sghai) [inline tests/foreman/cli/test_discoveryrule.py]: Typo in key name

File: tests/foreman/cli/test_hostgroup.py [hunk 2]
```diff
@@ -249,7 +249,6 @@ def test_positive_create_with_orgs_and_lce(self):
         hostgroup = make_hostgroup({
             'organization-ids': [org['id'] for org in orgs],
             'lifecycle-environment-id': lce['id'],
-            'organization-id': orgs[0]['id']
```
Comment (sghai) [inline tests/foreman/cli/test_hostgroup.py]: no such option `organization-id`

**Comment by oshtaier** [comment]
ACK pending test results

**Comment by sghai** [comment]
Test Results:

```console
robottelo]$ py.test -v tests/foreman/cli/test_discoveryrule.py::DiscoveryRuleTestCase -k test_negative_create_with_too_long_limit
================================================================ test session starts =================================================================
platform linux2 -- Python 2.7.12, pytest-3.0.2, py-1.4.31, pluggy-0.3.1 -- /usr/bin/python
cachedir: .cache
rootdir: /home/sghai/QE/robottelo, inifile: 
collected 25 items 

tests/foreman/cli/test_discoveryrule.py::DiscoveryRuleTestCase::test_negative_create_with_too_long_limit <- robottelo/decorators/__init__.py PASSED

================================================================ 24 tests deselected =================================================================
====================================================== 1 passed, 24 deselected in 36.67 seconds ======================================================


robottelo]$ py.test -v tests/foreman/cli/test_hostgroup.py::HostGroupTestCase -k test_positive_create_with_orgs_and_lce

================================================================ test session starts =================================================================
platform linux2 -- Python 2.7.12, pytest-3.0.2, py-1.4.31, pluggy-0.3.1 -- /usr/bin/python
cachedir: .cache
rootdir: /home/sghai/QE/robottelo, inifile: 
collected 21 items 

tests/foreman/cli/test_hostgroup.py::HostGroupTestCase::test_positive_create_with_orgs_and_lce PASSED

================================================================ 20 tests deselected =================================================================
====================================================== 1 passed, 20 deselected in 41.09 seconds ======================================================
```

**Comment by oshtaier** [comment]
ACK

**Comment by svtkachenko** [comment]
ACK

**Pull request merged**